2019/04/2019-04-23-bilibili.md

```diff
@@ -22,9 +24,10 @@ The following is our contact information:
 -   Company Address: [private]   
 -   Website: https://www.bilibili.com/
 
-I have a good faith belief that use of the copyrighted materials described above on the infringing web pages is not authorized by the copyright owner, or its agent, or the law. I have taken fair use into consideration.
+I am in good faith believing that the use and distribution of aforementioned copyrighted materials on the infringing repositories were not authorized by the copyright owner, its agent, or the law. I have taken fair use into consideration.
```

Comment:
I believe this sentence has no problem whatsoever, as this is the exact copy from the [*Guide to Submitting a DMCA Takedown Notice*](https://help.github.com/en/articles/guide-to-submitting-a-dmca-takedown-notice).

> Include the following statement: "I have a good faith belief that use of the copyrighted materials described above on the infringing web pages is not authorized by the copyright owner, or its agent, or the law. I have taken fair use into consideration."

> Also include the following statement: "I swear, under penalty of perjury, that the information in this notification is accurate and that I am the copyright owner, or am authorized to act on behalf of the owner, of an exclusive right that is allegedly infringed."

Reply:
Yeh nah mate I don't really care😂